src/components/page/meetingList/Card/MobileSize/index.tsx

```diff
@@ -0,0 +1,21 @@
+import { MeetingResponse } from '@api/meeting';
+import CardType from './CardType';
+import ListType from './ListType';
+
+export interface MobileSizeCardProps {
+  meetingData: MeetingResponse;
+  isAllParts: boolean;
+  mobileCardType: 'list' | 'card';
```

Comment:
```suggestion
  mobileType: 'list' | 'card';
```
요거 그냥 모바일 타입으로만 이름 지어도 좋을 것 같아요~!

Reply:
mobileType으로 통일해서 반영했어! 고마워~!